### ISSUE TITLE
repoduction of results on sintel

### ISSUE THREAD

**Issue by 16lemoing** [mulns/AccFlow#3]
Hello, thanks again for the great work!

I am interested in reproducing your results on sintel.
Could you share your trained models and testing script like you did for CVO dataset?

Also the train test splits are not clear to me. It seems that some videos are filtered out:
https://github.com/mulns/AccFlow/blob/23777724d4fbcd6c95a7817402a479a129c1b0ea/data/dataset.py#L174-L174
Instructions on how to prepare the data to match your setting would be very useful.

**Comment by mulns** [comment]
Certainly, I'm currently focused on another project, and I plan to release the associated code after November 18. It's worth noting that the High-Speed-Sintel dataset has been re-rendered, resulting in a misalignment of some videos with the original MPS-Sintel dataset. Consequently, we have chosen to exclude these videos to ensure a fair and equitable comparison.
